### DISCUSSION TITLE
Room temperature overshooting and settling too high

### DISCUSSION THREAD

**Discussion by dizzydnallov** [jmcollin78/versatile_thermostat#266] [General]
I love this add-on, and it works great for most of my rooms, but I have two rooms where overshooting occurs and then settles too far from the set temperature.  Attached is my kitchen, where the temperature is set at 20.5 but is achieving 22, then settles at 22.5 - you can see the "Regulated Temperature" does not drop below 20.  I have the "Self-Regulation" set to Low, but I am not sure if that makes any difference when overshooting(?)

Am I missing something that I can do - should the "Regulated Temperature" not try to compensate for this?

![Screenshot 2023-12-11 110512](https://github.com/jmcollin78/versatile_thermostat/assets/105526388/7cde24cc-0583-4d07-b877-c3b53dc26908)

**Comment by jmcollin78** [comment]
Hello, for this kind of question, I need, the VTherm attributes (Development Tool / States in formatted yaml with the <> in the editor) and regulation curves like described here: https://github.com/jmcollin78/versatile_thermostat/blob/main/README.md#even-better-with-plotly-to-tune-your-thermostat

Waiting for that you can try the Strong mode of regulation.
